prow/bugzilla/client.go

```diff
@@ -161,13 +161,15 @@ func (c *client) UpdateBug(id int, update BugUpdate) error {
 	return err
 }
 
+// CreateBug creates a new bug on the server.
+// https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#create-bug
 func (c *client) CreateBug(bug *BugCreate) (int, error) {
 	logger := c.logger.WithFields(logrus.Fields{methodField: "CreateBug", "bug": bug})
 	body, err := json.Marshal(bug)
 	if err != nil {
 		return 0, fmt.Errorf("failed to marshal create payload: %v", err)
 	}
-	req, err := http.NewRequest(http.MethodPost, fmt.Sprintf("%s/rest/bug/", c.endpoint), bytes.NewBuffer(body))
```

Comment:
Why didn't this require a change in the unit tests for the client?

Reply:
It did. Fixed in one of the force pushes I made trying to fix the bazel tests.